lms/services/lti_outcomes.py

```diff
@@ -110,7 +103,7 @@ class LTIOutcomesClient:
     XML_NS = "http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0"
 
     def __init__(self, _context, request):
-        pass
+        self.oath1_service = request.find_service(name="oauth1")
```

Comment:
```suggestion
        self.oauth1_service = request.find_service(name="oauth1")
```

Reply:
I have no idea why I do that, but I do it every time...